services/dashboard/search.rb

```diff
@@ -0,0 +1,29 @@
+class DashboardService
+  class Search < ApplicationService
+    def initialize(params)
+      @departure_id   = params[:departure_id]
+      @arrival_id     = params[:arrival_id]
+    end
+
+    def call
+      found_routes = Route.where(
+        start_id:       @departure_id,
+        destination_id: @arrival_id
```

Comment:
Extract to a separate method and avoid using instance variables
Something like this would look nice
```ruby
def call
  routes.map { |route| present(route) }
end

def routes
  @routes ||= Route.where(...)
end
```
Just an FYI - you can use `routes.map(&method(:present))` but it's slower and many people don't know about it